main.py

```diff
@@ -11,6 +12,11 @@
 my_user_id = getenv('MY_USER_ID')
 tweet_id = getenv('TWEET_ID')
 
+parser = argparse.ArgumentParser()
+parser.add_argument("-s", "--skip_following", help="Skip accounts that you have followed")
```

Comment:
I recommend using Argparse's `BooleanOptionalAction` with a `True` (?) default for better usability.